### PR TITLE
Guideline ios

### PR DESCRIPTION
**Pull request by nguoithichkhampha**
add guideline for iOS

### CODE REVIEW

File: ios/README.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+
+## Prerequisites
+- iOS device (iOS version 11.0 or higher)
```
Comment (anhdv282) [inline ios/README.md:3]: Did we test it with iOS 11 ?

File: ios/README.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+- EMOTIV App for iOS
+- Emotiv headset (if you don't have, you can purchase one [here](https://www.emotiv.com))
+- Emotiv account (if you don't have, you can register [here](https://www.emotiv.com))
+- The client id/secret of your cortex app (if you don't have, you can register [here](https://www.emotiv.com/my-account/cortex-apps/))
+
```
Comment (anhdv282) [inline ios/README.md:8]: the example has a property for 'license', please add a guide for getting a license if the user needs it.

File: ios/README.md (hunk 1 above)
Comment (nguoithichkhampha) [reply ios/README.md:3]: config build limit for iOS 11 or higher. We can not test on the all os version we support.

File: ios/README.md (hunk 2 above)
Comment (nguoithichkhampha) [reply ios/README.md:8]: ok, good suggestion.

File: ios/README.md (hunk 1 above)
Comment (anhdv282) [reply ios/README.md:3]: ok

**Pull request closed without merging**